Java/JavaNotebookApp/build.gradle.kts

```diff
@@ -12,6 +12,8 @@ repositories {
 dependencies {
     testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.1")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.8.1")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.8.9")
+    implementation("io.vertx:vertx-core:3.5.3")
```

Comment:
Please separate into two blocks: 1) implementations and 2) testImpl and RuntimeOnly block separated by a new line. Further, I usually sort such lists alphabetically.